docs/unlocked_package.rst

```diff
@@ -1,14 +1,98 @@
+
+If a package version already exists with the exact same contents, as determined by the hash of the package metadata, its ``SubscriberPackageVersion`` ID is returned rather than creating a new version.
+
+
+
+Handle Dependencies
+---------------------
+
+CumulusCI tries to convert dependencies configured under the ``project`` section of the ``cumulusci.yml`` file into a ``SubscriberPackageVersion`` ID (``04t`` key prefix). This format is required for dependencies in the API to create a package version.
+
+For dependencies that are specified as a managed package namespace and version, or dependencies specified as a GitHub repository with releases that can be resolved to a namespace and version, CumulusCI needs a scratch org with the dependencies installed to execute this conversion in the ``create_package_version`` task.
+
+.. important:: Install the dependencies in this scratch org *before* running the ``create_package_version`` task! 
+
+The scratch org's definition file also specifies the correct org shape when building the new package version.
+
+For dependencies that are an unpackaged bundle of metadata, CumulusCI creates an additional unlocked package to contain them.
+
+
+
+Promote a Package Version
 -------------------------
 
-Install the unlocked package
+To be installed in a production org, an unlocked package version must be `promoted <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_pkg_ver_promote.htm>`_ to mark it as released.
+
+CumulusCI does not yet provide tools to help promote an unlocked package version, so for now use the ``sfdx force:package:version:promote`` command. If additional unlocked packages were created to hold unpackaged dependencies, they must be promoted as well.
+
+
+
+Install the Unlocked Package
 ----------------------------
+
+There are two methods that can be used to install an unlocked package with CumulusCI.
+You can configure the ``update_dependencies`` and ``install_managed`` tasks to use a specific unlocked package by providing the 04t Id of the desired package.
+
+For the ``update_dependencies`` task this would be:
+
+.. code-block::console
+
+    $ cci run task update_dependencies --dependencies [{'version_id': '04t000000000'}]
+
+Or configured in ``cumulusci.yml`` like:
+
+.. code-block::yaml
+
+    task: update_dependencies
+    options:
+        dependencies:
+            - version_id: 04t000000000
+
+For the ``install_managed`` task this would be:
+
+.. code-block::console
+
+    $ cci task run intsall_managed --version 04t000000000
+
+Or configured in ``cumulusci.yml``:
+
+.. code-block::yaml
+
+    task: install_managed
+    options:
+        version: 04t000000000
+
```

Comment:
@davidmreed - Can you check me on these examples? I'm not sure about the `--dependencies` option on `update_dependencies` to be specific.